web/wp-content/themes/dctx-v2/template-parts/content.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-fpukvk.php, but found phpcs-scan-fPuKVK.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).